code/_onclick/hud/plane_master.dm

```diff
@@ -39,6 +39,18 @@
 /atom/movable/screen/plane_master/proc/clear_filters()
 	filters = list()
 
+///Blur rendered on z-levels below.
+/atom/movable/screen/plane_master/openspace_blur
+	name = "open space blur plane master"
+	plane = OPENSPACE_PLANE
+	appearance_flags = PLANE_MASTER
+	blend_mode = BLEND_OVERLAY
+	alpha = 255
+
+/atom/movable/screen/plane_master/openspace_blur/Initialize()
+	. = ..()
+	filters += filter(type = "blur", size = 0.75)
```

Comment:
don't manually use the filter proc, it's shit and dumb and bad. We have an api sitting on top of it, use add_filter() instead.

Reply:
Wait for Filterrific refactors to be merged so that relevant helper procs are available.

Reply:
you added more helpers to the filter api?

Reply:
I changed it to use `add_filter()`